README.md

```diff
@@ -19,6 +25,26 @@ PID   USER     TIME  COMMAND
 
 ## Usage
 To use this as your base image, construct your Dockerfile like so:
+For the latest version (openjdk-8u191-jre-alpine3.9-1.0):
+```
+FROM hmcts/cnp-java-base:openjdk-jre-8-alpine-2.0
+
+# Note: replace with your application name
+ENV APP moj-rhubarb-recipes-service.jar
+
+# Optional
+ENV JAVA_OPTS ""
+
+COPY build/libs/$APP /opt/app/
+
+# Note: replace with your app name. Cannot use $APP here as there is no shell expansion
```

Comment:
why not remove $APP then it's only used in one place, confusing to have to add it and then add this comment